boards/samr21-xpro/include/periph_conf.h

```diff
@@ -77,6 +81,58 @@ extern "C" {
 #define UART_1_PINS
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define PWM_NUMOF           (PWM_0_EN + PWM_1_EN)
+#define PWM_0_EN            1
+#define PWM_1_EN            1
+#define PWM_MAX_CHANNELS    2
+/* for compatibility with test application */
+#define PWM_0_CHANNELS      PWM_MAX_CHANNELS
+#define PWM_1_CHANNELS      PWM_MAX_CHANNELS
+
+/**
+ * @brief PWM channel configuration data structure
+ *
+ * TODO: this should be moved into the CPU folder
+ */
+typedef struct {
+    PortGroup *port;            /**< GPIO port */
+    uint8_t pin;                /**< GPIO pin */
+    uint8_t fnct;               /**< pin function multiplex value */
+    uint8_t chan;               /**< TCC channel to use */
+} pwm_conf_chan_t;
+
+/**
+ * @brief PWM device configuration data structure
+ *
+ * TODO: this should be moved into the CPU folder
+ */
+typedef struct {
+    Tcc *dev;                   /*< TCC device to use */
+    pwm_conf_chan_t chan[2];    /**< channel configuration */
+} pwm_conf_t;
+
+/* PWM device configuration */
+#if PWM_NUMOF
+static const pwm_conf_t pwm_config[] = {
+#if PWM_0_EN
+    {TCC1, {
+        {(PortGroup *)0x41004400, 6, 4, 0}, /* port 0, pin 6, af E, chan 0 */
+        {(PortGroup *)0x41004400, 7, 4, 1}  /* port 0, pin 7, af E, chan 1 */
+    }},
+#endif
+#if PWM_1_EN
+    {TCC0, {
+        {(PortGroup *)0x41004400, 18, 5, 2},
+        {(PortGroup *)0x41004400, 19, 5, 3}
```

Comment:
true. Best would be to find a generic pin mapping that fits all. I started a pin mapping document we can use to discuss:
https://docs.google.com/spreadsheets/d/1350pEm6-6vR7L_JGR5rHhaNbMrZgzkCgA-OS0akPH6U/edit#gid=1766063117